### ISSUE TITLE
Download Notecards and scripts

### ISSUE THREAD

**Issue by GoogleCodeExporter** [Animeimp/slmiv#6]
```
Feature request for SLMIV. When connecting to SL with SL Connection, what
the ability to automatically download all the text from the Notecards
and/or Scripts and store them locally.


```

Original issue reported on code.google.com by `joesox` on 23 Jan 2008 at 10:23

**Comment by GoogleCodeExporter** [comment]
```
How about retrieving image files. They should already be in cache?
```

Original comment by `laura18s...@gmail.com` on 4 Mar 2008 at 6:26

**Comment by GoogleCodeExporter** [comment]
```
To display images from SL cache takes third-party software that isn't that 
stable.
The images are compressed and in non-standard formats.  What I will probably do 
is
offer users a chance to download images,textures, scripts, and notecards, 
similar to
SLEEK (but SLeek v0.2.0 doesn't download them).
For more info about texture cache
https://wiki.secondlife.com/wiki/Texture_cache
```

Original comment by `joesox` on 4 Mar 2008 at 2:38

**Comment by GoogleCodeExporter** [comment]
```
This issue has been FIXED in version 2.10.16
```

Original comment by `joesox` on 23 Jan 2009 at 7:44
- Changed state: **Verified**
